repo/amg2023/package.py

```diff
@@ -31,7 +33,8 @@ class Amg2023(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("caliper", when="+caliper")
     depends_on("adiak", when="+caliper")
     depends_on("hypre+caliper", when="+caliper")
-    depends_on("hypre@2.31.0:")
+    # depends_on("hypre@2.31.0:")
```

Comment:
@gracenansamba It would be good to move the amg2023 changes to its own PR 
https://github.com/LLNL/benchpark/pull/318